react/modules/interpolateLink.ts

```diff
@@ -1,9 +1,10 @@
 import { getMappingFn, AvailableContext } from './mappings'
+import { Context } from './useInterpolatedLink'
```

Comment:
This is adding a circular dependency.

Also it should be imported as 

```suggestion
import type { Context } from './useInterpolatedLink'
```